src/plugins/jingleNodes/src/java/org/jinglenodes/JingleNodesPlugin.java

```diff
@@ -40,6 +40,9 @@
 import org.xmpp.jnodes.nio.PublicIPResolver;
 
 public class JingleNodesPlugin implements Plugin {
+	
+	private static final String[] 	stun_host = 	{"stun.l.google.com",	"stun.schlund.de",	"stun.iptel.org ",	"stun4.l.google.com"};
```

Comment:
I would prefer to see configuration properties for things like public network addresses.